src/Action.js

```diff
@@ -138,12 +142,21 @@ class Action {
    * @returns {Array<string>}
    */
   encodeForRecipe() {
-    return [
-      this._encodeForCall()[0],   // actionCallData
-      "0x0000000000000000000000000000000000000000000000000000000000000000",                        // subData
-      this.getId(),              // actionIds
-      this._getArgumentMapping(), // paramMappings
-    ]
+    if (CONFIG.chainId === 1) {
```

Comment:
if and else branches are the same?

Reply:
There was a time where they did diff. things, but that time has long passed, will remove the branch